packages/react-components/src/components/xy-output-component.tsx

```diff
@@ -426,24 +433,90 @@ export class XYOutputComponent extends AbstractTreeOutputComponent<AbstractOutpu
         }
     }
 
+    private tooltip(x: number, y: number): void {
+        this.isComputing = true;
+        const startPixel = this.positionXMove;
+        const points: any = [];
+        let zeros = 0;
+        this.state.xyData.datasets.forEach((d: any) => {
+            if (this.state.xyData.datasets.length === 1 || d.data[startPixel] > 0) {
+                const rounded = isNaN(d.data[startPixel]) ? 0 : (BIMath.round(Number(d.data[startPixel]) * 100)/BIMath.round(100));
+                const point: any = {
+                    label: d.label,
+                    color: d.borderColor,
+                    background: d.backgroundColor,
+                    value: rounded.toString(),
+                };
+                points.push(point);
+            }
+            else {
+                zeros += 1;
+            }
+        });
+
+        const tooltipData = {
+            title: this.state.xyData.labels[startPixel],
```

Comment:
Unfortunately we cannot use the labels from the data model, as these are currently rounded-out integers.

I think we need to compute the timestamp bigint from the mouse position. And the timestamp should be formatted using the numberTransalator.

Reply:
Done.